components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -274,27 +238,28 @@ public void variantsSetup(Set<String> enabledFields) throws Exception
         String sectionName = "variants";
         String[] fieldIds =
             { "variants", "variants_protein", "variants_transcript", "variants_dbsnp", "variants_zygosity",
-            "variants_effect", "variants_interpretation", "variants_inheritance", "variants_evidence",
-            "variants_segregation", "variants_sanger", "variants_coordinates" };
+                "variants_effect", "variants_interpretation", "variants_inheritance", "variants_evidence",
```

Comment:
It's OK to use it everywhere, but you can skip these changes when doing a specific pull request.